task_manager/tests/test_user.py

```diff
@@ -0,0 +1,238 @@
+import pytest
+from django.template.response import TemplateResponse
+from django.test import Client
+from freezegun import freeze_time
+from task_manager.tests.paths import resolve_path
+from task_manager.users.forms import ServiceUserForm
+from task_manager.users.models import ServiceUser
+
+pytestmark = pytest.mark.django_db
+
+
+def test_users_list(client: Client, service_user: ServiceUser) -> None:
+    # given
+    route = "/users/"
+
+    # when
+    response = client.get(route)
+
+    # then
+    expected_html = resolve_path("fixtures/users.html")
+    assert response.content.decode() == expected_html.read_text()
+    assert response.status_code == 200
+
+
+def test_create_user_get(client: Client) -> None:
+    # given
+    route = "/users/create/"
+
+    # when
+    response = client.get(route)
+    csrf_token = response.context.get("csrf_token")
+    form = ServiceUserForm()
+    context = {"form": form, "csrf_token": csrf_token}
+
+    # then
+    expected_html = TemplateResponse(
+        response.wsgi_request, "create_user.html", context=context
+    ).render()
+    assert expected_html.content == response.content
+    assert response.status_code == 200
+
+
+@freeze_time("15.07.2024 17:00")
+def test_create_user_post(client: Client) -> None:
+    # given
+    route = "/users/create/"
+    form_data = {
+        "username": "username",
+        "password1": "password",
+        "password2": "password",
+    }
+
+    # when
+    response = client.post(route, data=form_data)
+
+    # then
+    assert ServiceUser.objects.count() == 1
```

Comment:
wdyt, does it make sense to check that user was created with the right username?